.eslintrc.js

```diff
@@ -27,6 +27,14 @@
         "airbnb",
     ],
     "rules": {
+        "no-new": [0],
+        "class-methods-use-this": [0],
+        "no-restricted-properties": [0, {
```

Comment:
I believe I already merged such changes? Do you need to merge develop into your branch?

Reply:
I needed to do it in order to write unit tests. Probably small merge conflict will become here later, but I suppose it isn't big problem, to resolve it.